src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/DelegateCtorSignature.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+using Internal.TypeSystem;
+
+using Internal.Text;
+
+namespace ILCompiler.DependencyAnalysis.ReadyToRun
+{
+    public class DelegateCtorSignature : Signature
+    {
+        private readonly ReadyToRunCodegenNodeFactory _factory;
+
+        private readonly TypeDesc _delegateType;
+
+        private readonly ModuleToken _delegateTypeToken;
+
+        private readonly IMethodNode _targetMethod;
+
+        private readonly ModuleToken _targetMethodToken;
+
+        public DelegateCtorSignature(
+            ReadyToRunCodegenNodeFactory factory,
+            TypeDesc delegateType, 
+            ModuleToken delegateTypeToken,
+            IMethodNode targetMethod,
+            ModuleToken targetMethodToken)
+        {
+            _factory = factory;
+            _delegateType = delegateType;
+            _delegateTypeToken = delegateTypeToken;
+            _targetMethod = targetMethod;
+            _targetMethodToken = targetMethodToken;
+        }
+
+        protected override int ClassCode => 99885741;
+
+        public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
+        {
+            ObjectDataSignatureBuilder builder = new ObjectDataSignatureBuilder();
+            builder.AddSymbol(this);
+
+            if (relocsOnly)
+            {
+                builder.EmitReloc(_targetMethod, RelocType.IMAGE_REL_BASED_REL32);
```

Comment:
Nit: Since we aren't actually reloc'ing to this dependency in GetData, this is a good candidate to put in an override of `ComputeNonRelocationBasedDependencies`

Reply:
Good point, fixed in 3rd commit.